src/components/TodoList/TodoList.tsx

```diff
@@ -1,22 +1,84 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { Todo } from '../../types/Todo';
+import classNames from 'classnames';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
+  todos: Todo[];
+  onTodo: (u: Todo) => void;
+  onCurrentIcon: (v: HTMLElement | null) => void;
+}
 
-    <tbody>
-      <tr data-cy="todo" className="">
+export const TodoList: React.FC<Props> = ({ todos, onTodo, onCurrentIcon }) => {
+  const iconRefs = useRef<{ [key: number]: HTMLElement | null }>({});
```

Comment:
Using `iconRefs` for direct DOM manipulation is not recommended in React. Consider using state to manage class changes instead of directly manipulating the DOM.